### PR TITLE
Small author changes

### PR DESCRIPTION
**Pull request by Anri-Lombard**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.
  
  
  Simply added a small fix for Darren and a highlight to Andi that he's too humble to put up himself 🙂

### CODE REVIEW

**Comment by Anri-Lombard** [comment]
Simply added a small fix for Darren and a highlight to Andi that he's too humble to put up himself 🙂

(Comment seems out of sight, so just recommending it here)

File: content/authors/authors.ts [hunk 1]
```diff
@@ -117,7 +117,7 @@ export const Authors: Author[] = [
     photo: 'andi',
     name: 'Andi Qu',
     title: 'Co-Founder',
-    blurb: 'Andi Qu is an IOI and IMO bronze medalist.',
+    blurb: 'Andi Qu is an IOI and IMO bronze medalist, and part of the MIT Class of 2025',
```
Comment (SansPapyrus683) [inline content/authors/authors.ts:120]: is this needed

Comment (Aadit-Ambadkar) [reply content/authors/authors.ts:120]: benq has it, I don't see why andi cant...

Comment (Anri-Lombard) [reply content/authors/authors.ts:120]: I mean, technically it's appealing and attracts young coders.

If not we can remove it and focus on the small " error 🤷🏼‍♂️

Comment (envyaims) [reply content/authors/authors.ts:120]: doesnt hurt to have it no cap
it creates the imagery "cp can get me into mit!??!?!!? no way allow me to grind cp from now on"

**Review by SansPapyrus683 — APPROVED** [review]

File: content/authors/authors.ts (hunk 1 above)
Comment (ryanchou-dev) [reply content/authors/authors.ts]: wwait isn't up to andi if he wants to include it?

**Pull request merged**